common/src/main/scala/org/apache/celeborn/common/protocol/message/ControlMessages.scala

```diff
@@ -360,6 +362,10 @@ object ControlMessages extends Logging {
         .build()
   }
 
+  case class RemoveWorkersUnavailableInfo(
```

Comment:
Currently it's preferred not to define case classes for control messages, use pbs instead.